src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -20,13 +20,19 @@ public class Ado : NotificationsBase, IAdo {
 
     public async Async.Task<OneFuzzResultVoid> NotifyAdo(AdoTemplate config, Container container, IReport reportable, bool isLastRetryAttempt, Guid notificationId) {
         var filename = reportable.FileName();
-        if (reportable is RegressionReport) {
-            _logTracer.LogInformation("ado integration does not support regression report. container:{Container} filename:{Filename}", container, filename);
-            return OneFuzzResultVoid.Ok;
+        Report? report;
+        if (reportable is RegressionReport regressionReport) {
+            if (regressionReport.CrashTestResult.CrashReport is not null) {
+                report = regressionReport.CrashTestResult.CrashReport;
+                _logTracer.LogInformation("parsing regression report for ado integration. container:{Container} filename:{Filename}", container, filename);
+            } else {
+                _logTracer.LogInformation("ado integration does not support this regression report. container:{Container} filename:{Filename}", container, filename);
```

Comment:
We probably want this at a higher log level?

Reply:
Switched to LogError